### ISSUE TITLE
Certificate for request

### ISSUE THREAD

**Issue by axi92** [filipedeschamps/rss-feed-emitter#212]
Is there an option to pass an certificate to the request?

**Comment by TobiTenno** [comment]
Not at present. What kind of certificate are you looking to pass?

**Comment by axi92** [comment]
https://github.com/request/request#tlsssl-protocol
ca-cert
We use a self signed cert, so I need to pass the cert to the request.

**Comment by TobiTenno** [comment]
you'll probably want to look at how this would be done with `node-fetch`, i'm in progress attempting to convert from request to something still supported

**Comment by axi92** [comment]
I moved in all of my projects to https://www.npmjs.com/package/got where I used request.
Got is nice to use and behaves similar to request.

**Comment by TobiTenno** [comment]
Fair enough. I'm trying to use a more browser-friendly API, since node-fetch is the node impl of the web api of the same name. That's my motivation for that.
